Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -137,20 +137,23 @@ void WebsiteDataStore::forEachWebsiteDataStore(Function<void(WebsiteDataStore&)>
 
 Ref<WebsiteDataStore> WebsiteDataStore::createNonPersistent()
 {
-    return adoptRef(*new WebsiteDataStore(WebsiteDataStoreConfiguration::create(IsPersistent::No), PAL::SessionID::generateEphemeralSessionID()));
+    auto result = adoptRef(*new WebsiteDataStore(WebsiteDataStoreConfiguration::create(IsPersistent::No), PAL::SessionID::generateEphemeralSessionID()));
```

Comment:
Please use `Ref` instead of `auto`. We prefer not to hide smart pointers anymore.